### DISCUSSION TITLE
Message Logger

### DISCUSSION THREAD

**Discussion by authorisation** [rodit/SnapMod#272] [Ideas & Feature Suggestions]
A Feature that logs deleted messages and maybe also deleted snaps.

**Comment by RevealedSoulEven** [comment]
Bro bro bro.... That will only work, if snapchat is running in background, and your mobile data is on😂
Else it's useless.

**Reply by authorisation** [reply]
Oh yeah I didn't even think about that

**Comment by RevealedSoulEven** [comment]
Hue hue hue. Sad loife bro. And snapchat isn't working like whatsapp so
that it's end to end encrypted and we can see the deleted receipts. It
works on servers completely, so messages one deleted are not possible tk
see (maybe)

**Reply by authorisation** [reply]
Maybe you can prevent messages from disappearing after viewing them locally?

**Comment by RevealedSoulEven** [comment]
That will only work till you close the snapchat app. Once you restart it,
it will again load the messages from servers and it will be useless.

The dev might have already worked on it I'm sure

**Comment by RevealedSoulEven** [comment]
Well the thing you said message logger is quite useful.

Why?

See, some of our friends don't like saving messages and even they set chats to be deleted after viewing. In that case just as export option works exporting all msgs to txt file, same way message logger can save all out messages as soon as it is recieved or sent, and keeps saving it to a custom file with the conversation ID or whatever rodit likes to make it. But it will log all messages and appends them to a txt file.

I also thought of asking this too earlier, but rodit is quite busy so better, the current bugs are fixed that's enough for now. Once he gets free, I'll request more features one by one
